recipes/aws-cdi-sdk/all/conanfile.py

```diff
@@ -0,0 +1,110 @@
+import os
+import re
+
+from conans import AutoToolsBuildEnvironment, CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.35.0"
+
+class AwsCdiSdkConan(ConanFile):
+    name = "aws-cdi-sdk"
```

Comment:
The internal name is `aws-cpp-sdk-cdi` but this matches the repo 🤔 not sure which to match

Reply:
this package contains two libraries: aws-cpp-sdk-cdi and cdisdk which is a C library